ansible/roles/prosody/files/mod_muc_webhooks.lua

```diff
@@ -489,10 +491,6 @@ function handle_occupant_access(event, event_type)
             end
         end
 
-        if final_event_type == PARTICIPANT_JOINED then
-                payload.roomCreatedTimestamp = room.created_timestamp;
-        end
```

Comment:
We no longer need this?

Reply:
nope, i had to change the implementation